experimental/linting_tool/lib/pages/saved_lints_page.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2021 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class SavedLintsPage extends StatelessWidget {
+  const SavedLintsPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return const Text('Saved Profiles');
```

Comment:
I'm assuming this will fill out? Maybe a TODO that sketches what is intended for this file?

Reply:
Yup, I'll add TODOs.